content/en/events/2023/kcscn/registration.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Registration"
+type: docs
+weight: 2
+---
+
+Registration is open to all members of one of the [Kubernetes orgs] or
+developer that is planning to contirbute to Kubernetes.
```

Comment:
```suggestion
Registration is open to all members of one of the [Kubernetes orgs].
```

Nowadays, as we don't have New Contributor Workshop, it's limited to only members, the only exception would be a SIG sponsoring someone.